File: 1 ano/CP3 JAVA/src/CP3/Pessoa.java
package CP3;

public class Pessoa {
	private String nome;
	private String ra;
	
	Pessoa(){
		
	}
	
	Pessoa(String nome, String ra){
		this.nome = nome;
		this.ra = ra;
	}
	
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getRa() {
		return ra;
	}
	public void setRa(String ra) {
		this.ra = ra;
	}
}

File: C#/CP1/CP1-ClassLibrary/Models/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using CP1_ClassLibrary;


namespace CP1_ClassLibrary.Models
{
    public class Pessoa
    {
        private String nome;
        private String cpf;

        public Pessoa(string nome, string cpf)
        {
            this.nome = nome;
            this.cpf = cpf;
        }

        public Pessoa() : this("Gustavo", "123") {  }

        public String GetNome() { 
            return this.nome;
        }

        protected String GetCPF() { 
            return this.cpf; 
        }

        public void SetNome(String nome)
        {
            this.nome = nome;
        }

        protected void SetCPF(String cpf)
        {
            this.cpf = cpf;
        }
    }
}
